core/governance/engine.go

```diff
@@ -311,6 +313,10 @@ func (e *Engine) OnTick(ctx context.Context, t time.Time) ([]*ToEnact, []*VoteCl
 
 	now := t.Unix()
 
+	// use slice for deterministic behaviour and event order
+	// succeededMarkets := []string{}
```

Comment:
?

Reply:
Lots of comments in this PR left as things I want to work on/look in to further down the line. This is one of them